bmaptools/BmapHelpers.py

```diff
@@ -83,3 +94,72 @@ def program_is_available(name):
             return True
 
     return False
+
+
+def get_mount_point(filepath):
+    """
+    Return the mount point the file object file resides on.
+    """
+    f = os.path.realpath(filepath)
+    while f != os.path.sep:
+        if os.path.ismount(f): return f
+        f = os.path.realpath(os.path.join(f, os.pardir))
+    return f
+
+
+def get_file_system_type(filepath):
+    """
+    Return the file system type the file object file resides on.
+    """
+
+    # normalize,resolve and get absolute path to the file object
+    f = os.path.realpath(filepath)
+    proc = subprocess.Popen(['df', '-T', '--', f], stdout=PIPE, stderr=PIPE)
+    stdout, stderr = proc.communicate()
+    ftype = None
+    if stdout:
+        t = stdout.splitlines()
+        if len(t) >= 2: 
+            t = t[1].split(None, 2)
+            if len(t) >= 2: 
+                ftype = t[1].lower()
+
+    if not ftype:
+        raise Error("no file system type was found: %s" % stderr)
+    return ftype
+
+
+def get_zfs_compat_param_path():
+    """
+    Returns path to required zfs kernel compatibility param.
+    """
+
+    return '/sys/module/zfs/parameters/zfs_dmu_offset_next_sync'
```

Comment:
Remove the function please, it brings no value. If this path is used only once, have a local variable. Otherwise, introduce a global variable instead.